tracing-subscriber/tests/option.rs

```diff
@@ -0,0 +1,40 @@
+use tracing::level_filters::LevelFilter;
```

Comment:
these tests don't compile without the "registry" feature enabled: https://github.com/tokio-rs/tracing/runs/7141077462?check_suite_focus=true

let's add
```suggestion
#![feature = "registry"]
use tracing::level_filters::LevelFilter;
```